package.json

```diff
@@ -9,7 +9,8 @@
     "coverage": "npm run unit -- --coverage-report=text-summary",
     "postcoverage": "tap --coverage-report=lcov --no-browser",
     "lint": "eslint . --ignore-path=.gitignore",
-    "test": "npm run lint && npm run unit && npm run checkonly",
+    "test": "npm run lint && npm run unit && npm run checkonly && npm run typescript",
+    "typescript": "tsc --project ./tests/types/tsconfig.json",
```

Comment:
We should add the types field [as in this link](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package)